exercises/09_dedup/main_test.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import "testing"
+
+func equal(a, b string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i := range a {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+func TestDedup(t *testing.T) {
+	for _, tc := range []struct {
+		name  string
+		input string
+		want  string
+	}{
+		{"empty", "", ""},
```

Comment:
Fails for 
```go
{"leading ascii 0", "\x00", "\x00"},
```

Reply:
![image](https://user-images.githubusercontent.com/65840/147662743-dc782dee-1bee-4c76-b5c1-3be009ec7055.png)